presto-main/src/test/java/com/facebook/presto/sql/planner/TestDomainTranslator.java

```diff
@@ -156,6 +154,26 @@
     private static final long COLOR_VALUE_1 = 1;
     private static final long COLOR_VALUE_2 = 2;
 
+    private Metadata metadata;
+    private LiteralEncoder literalEncoder;
+    private DomainTranslator domainTranslator;
+
+    @BeforeClass
+    public void setup()
+    {
+        metadata = createTestMetadataManager();
```

Comment:
Perhaps add to cmt msg that this doesn't change anything, since non-static `metadata, literalEncoder, domainTranslator` are init-ed the very same way they used to be.